alsa_settings/LNLM_SDW_AIOC.sh

```diff
@@ -4,6 +4,10 @@ set -e
 amixer -c sofsoundwire cset name='Headphone Switch' on
 amixer -c sofsoundwire cset name='rt711 FU05 Playback Volume' 80
 
+# override jack detection mode to headset
+# related linux pr: https://github.com/thesofproject/linux/pull/4969
+amixer -c sofsoundwire ct name='rt711 GE49 Selected Mode' 2 || true
```

Comment:
is this really 'ct' or a typo where 'cset' was intended ? Not sure I follow this @fredoh9

Reply:
oops. copy & paste and unwanted input was added. Will fix this.